sales.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <meta charset="UTF-8">
+  <title>Sales</title>
+  <nav>
+    <ul id="navigation">
+      <li><a href='file:///Users/codyhulsey/projects/cookie_stand_201/mainpage.html'>HOME</a></li>
+      <li><a href="file:///Users/codyhulsey/projects/cookie_stand_201/index.html">SALES</a></li>
+      <li>
+      <a href="file:///Users/codyhulsey/projects/cookie_stand_201/styleguide.html">STYLE GUIDE</a></li>
+    </ul>
+  </nav>
+</head>
+<body>
+  <form id="myForm">
+    <fieldset>
+      <legend>Create your own store</legend>
+
+      <label for="storeName">Store Name</label>
+      <input type="text" id='storeName'></input>
+  </fieldset>
+
```

Comment:
It's nice for readability but HTML should be compact so you can begin to eliminate the unused line breaks any time.